doc/api/cli.md

```diff
@@ -51,6 +51,28 @@ entry point, the `node` command will accept as input only files with `.js`,
 [`--experimental-wasm-modules`][] is enabled; and with no extension when
 [`--experimental-default-type=module`][] is passed.
 
+## Sub-commands
+
+### `node run [command]`
```

Comment:
`node inspect` is an existing subcommand. I'm surprised a new section is needed.

Reply:
`node inspect` is only mentioned in `Synopsis` and there is no specific section for `inspect` in `cli.md`